tests/tls_e2e/server_enc/CMakeLists.txt

```diff
@@ -44,6 +44,6 @@ if (BUILD_OPENSSL)
 
   enclave_include_directories(openssl_tls_server_enc PRIVATE
                               ${CMAKE_CURRENT_BINARY_DIR})
-
+                              
```

Comment:
Remove this noop change. (This kind of change should get caught by trim trailing whitespace tools e.g. https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut)